src/artemis/tests/test_zocalo_interaction.py

```diff
@@ -80,3 +82,41 @@ def test_run_start_and_end(
     function_to_run = partial(function_to_test, EXPECTED_DCID)
     function_to_run = partial(function_wrapper, function_to_run)
     _test_zocalo(function_to_run, expected_message)
+
+
+@patch("workflows.recipe.wrap_subscribe")
+@patch("zocalo.configuration.from_file")
+@patch("src.artemis.zocalo_interaction.lookup")
+def test_when_message_recieved_from_zocalo_then_point_returned(
+    mock_transport_lookup, mock_from_file, mock_wrap_subscribe
+):
+    message = {
+        "max_voxel": [3, 5, 5],
+        "centre_of_mass": [2.942925659754348, 7.142683401382778, 6.79110544979448],
+    }
+    step_params = {"dcid": "8183741", "dcgid": "7263143"}
```

Comment:
should - the values here which are also used further below (dcid and max_voxel) should be put into constants